clox/functions.py

```diff
@@ -136,18 +136,18 @@ def _get_weekday_id(first_weekday: str, date_system: str = "gregorian") -> int:
     if len(first_weekday) > 2:
         first_weekday_normalized = first_weekday_normalized[:2]
     weekdays = [x[:2] for x in WEEKDAYS_LIST]
-    if date_system == "jalali":
+    if date_system.upper() == "JALALI":
```

Comment:
Since we are upperccasing it in the `argaparese` input I'd say there's no need to `.upper()` it again.

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/62/commits/1363ea77161208dcd6437371edd00063eacb4a7d